README.md

```diff
@@ -1,24 +1,14 @@
 # README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Thanks for taking a look at this!
 
-Things you may want to cover:
+A couple of questions:
 
-* Ruby version
+* Is there any good way to add js to ransack?  Trying to get it to find without reloading.  I've heard re-rendering
+the contact loop on click could work.  What would that look like?
```

Comment:
The ransack form should work similarly to the other AJAX we've done. Since it uses the `search_form_for`, which is based off of [`form_for`](https://apidock.com/rails/ActionView/Helpers/FormHelper/form_for) rather than [`form_with`](https://apidock.com/rails/ActionView/Helpers/FormHelper/form_with), you need to add `remote: true` instead of `local: false` to enable AJAX on the form.

Then the rest of the steps should be the same:
- Update the `index` action to respond to JS requests
- Create `index.js.erb` file that removes the previous results and adds the new results to the page.

How does that sound?